db/migrate/20210104141058_create_timeline_entries.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTimelineEntries < ActiveRecord::Migration[6.0]
+  def change
+    create_table :timeline_entries do |t|
+      t.references :coronavirus_page, foreign_key: true
```

Comment:
Oh can you add a `null: false` to this? I thought that happened automatically but it hasn't. Aside from that this looks just about ready to merge.

Reply:
Cheers Kevin, think is should be there and in the schema now 🙂